smqtk_classifier/classify_descriptor_collection.py

```diff
@@ -283,3 +283,11 @@ def classify_arrays(
         for label, classifier in label2classifier.items():
             label2pred[label] = list(classifier.classify_arrays(array_seq))
         return label2pred
+
+
+class ClassifierCollection(ClassifyDescriptorCollection):
+    warn("ClassifierCollection was renamed to "
```

Comment:
This should be within the `__init__` constructor method otherwise this will just emit on import and not at the more useful time of attempted use.